integration/js/utils/SdkBaseTest.js

```diff
@@ -35,15 +35,17 @@ class SdkBaseTest extends KiteBaseTest {
     this.testReady = false;
     this.testFinish = false;
     this.testName = testName;
-    this.useSimulcast = !!this.payload.useSimulcast;
+    this.useSimulcast = this.payload?.useSimulcast || false;
+    this.namespaceInfix = this.payload?.namespaceInfix || '';
+
     if (this.useSimulcast) {
       this.testName += 'Simulcast';
     }
     this.useVideoProcessor = !!this.payload.useVideoProcessor;
     if (this.useVideoProcessor) {
       this.testName += 'Processor';
     }
-    this.capabilities["name"] = process.env.STAGE !== undefined ? `${this.testName}-${process.env.TEST_TYPE}-${process.env.STAGE}`: `${this.testName}-${process.env.TEST_TYPE}`;
+    this.capabilities["name"] = process.env.STAGE !== undefined ? `${this.testName}-${process.env.TEST_TYPE}-${process.env.STAGE}` : `${this.testName}-${process.env.TEST_TYPE}`;
```

Comment:
nit: ""-> ''

Reply:
Updated all double quotes to single quotes.